lib/cc/engine/analyzers/analyzer_base.rb

```diff
@@ -11,6 +11,10 @@ class Base
           ::RubyParser::SyntaxError,
         ]
 
+        DEFAULT_MASS_THRESHOLD = 28
```

Comment:
@wfleming Thanks for the tip about the thought behind the thresholds and parsing. I can investigate that. 

I reference the classic rubric you mentioned in [my commit](https://github.com/codeclimate/codeclimate-duplication/pull/61/commits), along with the Python source.

I've been keeping track of test repos [here](https://gist.github.com/ABaldwinHunter/e280310b4987766efd8c).

In general, it seemed that on classic, duplication and complexity issues had much greater grade impact than they did on Platform.

I tested the current duplication setup on `app` - but could do some further testing with this exact setup on additional repos.